sql/load_demo_data.sql

```diff
@@ -3178,6 +3312,10 @@ CREATE TABLE `oh_medicaldsrward` (
   KEY `FK_MEDICALDSRWARD_MEDICALDSR_idx` (`MDSRWRD_MDSR_ID`),
```

Comment:
The multiple step version for this table has:
<pre>
  `MDSRWRD_WRD_ID_A` char(3) NOT NULL DEFAULT '',
</pre>
where as the load demo data has:
<pre>
  `MDSRWRD_WRD_ID_A` char(3) NOT NULL,
</pre>